src/prefect/cli/agent.py

```diff
@@ -71,7 +71,7 @@ async def start(
         None,
         "-t",
         "--tag",
-        help="DEPRECATED: One or more optional tags that will be used to create a work queue",
+        help="DEPRECATED: One or more optional tags that will be used to create a work queue. This feature is planned to be fully deprecated on 2023-02-23.",
```

Comment:
We go from deprecated -> removed so this should read "This option will be removed on ..."